Classes/Controller/AbstractController.php

```diff
@@ -141,6 +141,11 @@ protected function loadDocument($requestData)
                         $this->document = GeneralUtility::makeInstance(Document::class);
                     }
 
+                    //it looks that for not loaded from database it is always 0, so needs to be set up on load
```

Comment:
```suggestion
                    // Make sure configuration PID is set when applicable
```
(see https://github.com/kitodo/kitodo-presentation/pull/803#discussion_r937722879)